teleop_twist_joy_node:
  ros__parameters:
    axis_linear:
      x: 1
    scale_linear:
      x: 0.3
    scale_linear_turbo:
      x: 0.6

    axis_angular:
      yaw: 0
    scale_angular:
      yaw: 0.4

    enable_button: 5  # L2 shoulder button
    enable_turbo_button: 0  # L1 shoulder button
    base_width: 0.52
    wheel_radius: 0.125
    deadzone: 0.1
    enable_track_control_button: 3